bedrock/products/tests/test_views.py

```diff
@@ -286,18 +286,43 @@ def test_simple_get__for_valid_locale_with_enough_content(self, render_mock):
             ],
         )
 
+    def test_simple_get__for_invalid_locale(self, render_mock):
```

Comment:
These tests were swapped in order, to make context easier when reading them all cold in the future